.gitlab-ci.yml

```diff
@@ -2,6 +2,8 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-commits.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-black-format-code.yml'
```

Comment:
By the way, you can temporary test this by setting a specific URL like:
```
include:
  - remote: 'https://raw.githubusercontent.com/oleorhagen/mendertesting/QA-131/.gitlab-ci-format-python3-code.yml'
```

Reply:
Smrt!